Icons Navbar on SectionED

c17alepe opened the pull request:
I have created and adjusted icons to replace the buttons in #7210

I now need feedback on what to improve. I tried Henriks tip about using the fist for ResultEd but it doesnt work as one color. Here is the results and some alternatives. 

Black lines doesnt work on this purple
![image](https://user-images.githubusercontent.com/38323901/58270642-93320b00-7d8a-11e9-8c84-0f4dd1b88ae3.png)

This inverted one looks to thin
![image](https://user-images.githubusercontent.com/38323901/58270645-975e2880-7d8a-11e9-98d9-d981d2334b3b.png)

This one looks okay but the other ones doesnt have out-lines, so it is not perfectly consistant with the other ones
![image](https://user-images.githubusercontent.com/38323901/58270983-4dc20d80-7d8b-11e9-886e-6b53d16c213a.png)



I think the last one looks bearable but id much rather have some suggestions what we could switch it with. I also of course want feedback on the other icons and its positioning. 

Past History:
#7296
#7295

Test site
https://c17alepe.webug.his.se:20002/DuggaSys/sectioned.php?courseid=2&coursevers=97732

c17alepe commented:
@HGustavs What do you think?

a17pongo commented:
looks great I think and seems more intuitive.
I would go for suggestions nr.2

c17johha commented on DuggaSys/sectioned.php:
```diff
@@ -86,24 +82,32 @@
 					<td class='access menuButton' style='display: inline-block;'>
 						<div class='access menuButton'>
-						<a id='accessBTN' href='' onclick='navigatePage(this.id, "accessed.php");' oncontextmenu='javascript:navigatePage(this.id, "accessed.php");'><input type='button' value='Access' class='submit-button' title='Give students access to the selected version'/></a>
+						<a id='accessBTN' title='Give students access to the selected version' value='Access' href='' onclick='navigatePage(this.id, "accessed.php");' oncontextmenu='javascript:navigatePage(this.id, "accessed.php");'>
```
Indentation police was here, Use 1 TAB

c17johha commented on DuggaSys/sectioned.php:
```diff
@@ -86,24 +82,32 @@
+					<td class='files' style='display: inline-block;'>
 						<div class='files menuButton'>
-						<a id='filesBTN' href='' onclick='navigatePage(this.id, "fileed.php");' oncontextmenu='javascript:navigatePage(this.id, "fileed.php");'><input type='button' value='Files' class='submit-button' title='Show files'/></a>
+						<a id='filesBTN' title='Show files' value='Files' href='' onclick='navigatePage(this.id, "fileed.php");' oncontextmenu='javascript:navigatePage(this.id, "fileed.php");'>
```
Indentation police was here, Use 1 TAB

c17johha commented on DuggaSys/sectioned.php:
```diff
@@ -86,24 +82,32 @@
+					<td class='results' style='display: inline-block;'>
 						<div class='results menuButton'>
-							<a id='resultsBTN' href='' onclick='navigatePage(this.id, "resulted.php");' oncontextmenu='javascript:navigatePage(this.id, "resulted.php");'><input type='button' value='Results' class='submit-button' title='Edit student results' /></a>
+							<a id='resultsBTN' title='Edit student results' value='Results' href='' onclick='navigatePage(this.id, "resulted.php");' oncontextmenu='javascript:navigatePage(this.id, "resulted.php");'>
```
Indentation police was here, Use 1 TAB for 87-89

c17alepe commented:
Fixed indentation
Added tooltip
Changed the fist icon
Added more margin between the icons.
Made hamburger menu have border-radius on all sides
Added subtle shadow hover effect to navButt.

The navbutt hover effect will now appear everywhere on the site. So i might have stepped on toes i shouldnt have now. Let me know if this is a bad change

c17alepe commented:
The searchbar gets a weird shadow effect now. Is there any other place as well?

c17johha commented on DuggaSys/sectioned.php:
```diff
@@ -44,18 +44,14 @@
-								<button type='button' class='submit-button no-radius' style='width:35px;margin-left:0px' title='Edit the selected version' onclick='showEditVersion();'>
-									<img id='versionCog' style='margin-top:6px' src='../Shared/icons/CogwheelWhite.svg'>
-								</button>
+								<img id='versionCog' class="navButt" title='Edit the selected version' onclick=showEditVersion(); src='../Shared/icons/CogwheelWhite.svg'>
```
Indentation appears wrong for me

b17nicsv commented:
Does every icon have a tooltip now?

c17alepe commented:
Yes, now they do.

c17johha commented:
@b17nicsv yes
